spring-cloud-dataflow-server/docker-compose-dood.yml

```diff
@@ -47,7 +47,7 @@ services:
 
   skipper-server:
     environment:
-      - SPRING_CLOUD_SKIPPER_SERVER_PLATFORM_LOCAL_ACCOUNTS_DEFAULT_DOCKER_NETWORK=${COMPOSE_PROJECT_NAME:?COMPOSE_PROJECT_NAME must be set when DooD is enabled}_default
+      - SPRING_CLOUD_SKIPPER_SERVER_PLATFORM_LOCAL_ACCOUNTS_DEFAULT_DOCKER_NETWORK=${COMPOSE_PROJECT_NAME:-scdf}_default
```

Comment:
This will not work! 
The `COMPOSE_PROJECT_NAME` is not a SCDF but internal docker-compose variable: https://docs.docker.com/compose/reference/envvars/#compose_project_name

The docker compose uses the `COMPOSE_PROJECT_NAME` to compute the network name. So to change the network name you must set the COMPOSE_PROJECT_NAME variable explicitly or via a supplementary property file (e.g https://docs.docker.com/compose/environment-variables/#the-env-file)

In the lines above we only reuse the docker compose's network (e.g. COMPOSE_PROJECT_NAME ) for the local deployers docker containers: SPRING_CLOUD_SKIPPER_SERVER_PLATFORM_LOCAL_ACCOUNTS_DEFAULT_DOCKER_NETWORK

Note that `COMPOSE_PROJECT_NAME:-scdf` unfortunately wont set the  COMPOSE_PROJECT_NAME variables for the docker-compose. IMO this limitation but the only workaround is to either set it explicitly (e.g. export COMPOSE_PROJECT_NAME=...) or have a companion properties file (e..g. https://docs.docker.com/compose/environment-variables/#the-env-file)